main.py

```diff
@@ -15,19 +14,18 @@
 PROJECT_FILE = ''
```

Comment:
the project file is given per commandline why still have this?

Reply:
Fixed with https://github.com/Squiddrone/VPParametricsSolver/pull/2/commits/4c16266a6748f824b112cf3f8e9ee007f8f268ff